valohai/distributed.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import List, Optional
+
+from valohai import paths
+from valohai.internals.distributed_config import DistributedConfig, Member
+from valohai.internals.distributed_config.consts import MASTER_MEMBER_ID
+
+
+class Distributed:
+    """
+    Distributed toolkit accessed through `valohai.distributed`.
+    """
+
+    _config: Optional[DistributedConfig] = None
+
+    def is_distributed_task(self) -> bool:
+        # not a property to mimic `is_running_in_valohai`
+        try:
+            return bool(self.config.group_name)
+        except FileNotFoundError:
+            return False
+        except json.decoder.JSONDecodeError:
+            return False
+
+    @property
+    def group_name(self) -> str:
+        return self.config.group_name
+
+    @property
+    def member_id(self) -> str:
+        return self.config.member_id
+
+    @property
+    def rank(self) -> Optional[int]:
+        return self.self().rank
+
+    @property
+    def required_count(self) -> int:
+        return self.config.required_count
+
+    def members(self) -> List[Member]:
+        return self.config.members
+
+    def member(self, member_id: str) -> Member:
+        for member in self.members():
+            if member.member_id == member_id:
+                return member
+        raise RuntimeError(f"No member with id {member_id}")
+
+    def self(self) -> Member:
+        return self.member(member_id=self.config.member_id)
+
+    def master(self) -> Member:
+        return self.member(member_id=MASTER_MEMBER_ID)
```

Comment:
I'd like to drop the whole `MASTER_MEMBER_ID` thing, so maybe
```suggestion
        for member in self.members():
            if member.is_master:
                return member
        raise RuntimeError("No master configured")
```

unless this is moved up to the `config`; `.master_member` sounds pretty nice...

Reply:
yeah, we could drop the `MASTER_MEMBER_ID`, will fix